src/test/java/useCases/impactor/StatusImpactorTest.java

```diff
@@ -0,0 +1,74 @@
+package useCases.impactor;
+
+import entity.*;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.BeforeAll;
+import useCases.StatusChecker;
+
+import java.util.HashMap;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class StatusImpactorTest {
+    private static Player player;
+    private static HashMap<String, Integer> initStatusMap;
+
+    /**
+     * setup function for this test
+     */
+    @BeforeAll
```

Comment:
Maybe we want to change this to
`@BeforeEach`
as well

So that all the tests are ran under the same environment, and won't conflict if the orders are different